docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -92,7 +92,8 @@ Analysis takes minutes to hours to run depending on the size of the application
 ====
 {ProductShortName} relies on Kubernetes scheduling capabilities to determine how many analyzer instances are created based on cluster capacity. If several applications are selected for analysis, by default, only one analyzer can be provisioned at a time. With more cluster capacity, more analysis processes can be executed in parallel.
 ====
-. When analysis is complete, click the *Report* link to see the results of the analysis.
-. Optional: Display the details of the analysis:
-.. Click the Options menu ({kebab}).
-.. Select *Analysis details*. You can choose either the YAML or JSON format.
```

Comment:
@mpershina This should be documented with changes. something like:
When analysis is complete, click the application name to open the application drawer and see the results of the analysis.

Reply:
Attempted to rewrite it a bit, see the preview: [Configuring and running an application analysis](https://mpershina.github.io/Previews/MTA/MTA-3110#mta-web-configuring-and-running-an-application-analysis_user-interface-guide).